governance/votes/Executive Vote - March 6, 2024.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Template - [Executive Vote] Executive Vote - March 6, 2024
```

Comment:
We typically include the items contained in the spell in the title for better searchability and consistency. Is there a reason for excluding them this time?